packages/databyss-ui/modules/Homepage/homepageContent.json

```diff
@@ -0,0 +1,101 @@
+{
+    "title": "Databyss",
+    "meta": [
+        {
+            "name": "description",
+            "content": "You research, dabble, take notes, and get lost in your thoughts. Databyss is your new word-processor."
+        }
+    ],
+    "sections": [
+        {
+            "component": "Hero",
+            "backgroundImgSrc": "/homepage/stone_bg.jpg",
+            "logoSrc": "/homepage/logo_new.png",
+            "title": "Databyss",
+            "headline": "You research, dabble, experiment, take notes, and get lost in your thoughts. Databyss is your new word-processor.",
+            "buttonText": {
+                "bold": "Sign up",
+                "normal": "(for free)"
+            },
+            "buttonHref": "https://app.databyss.org/signup",
+            "navLinks": [
+                {
+                    "name": "Home",
```

Comment:
We don't need the Home link here. However, we should like the icon on `Private.js` back to `www.databyss.org` so it goes to the homepage.